src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -47,7 +48,16 @@ public static function create(\Exception $exception, $statusCode = null, array $
             $statusCode = 500;
         }
 
+        switch ($statusCode) {
+            case 404:
+                $title = 'Sorry, the page you are looking for could not be found.';
```

Comment:
If you're proposing this feature for an API, I think the term "page" should be reworded. I really don't know which could be better: "route", "resource", "path", "endpoint", "URI"?